src/components/styled/StyledPagination.tsx

```diff
@@ -19,7 +19,7 @@ const StyledPagination = styled(Pagination)<PaginationProps>({
   "& .MuiPaginationItem-previousNext": {
     borderRadius: "0px 6px 6px 0px",
   },
-  "& li:first-child > button": {
+  "& li:first-of-type > button": {
```

Comment:
updated this selector so that it is SSR friendly as `first-child` can change between hydration